website/docs/docs/dbt-versions/core-upgrade/00-upgrading-to-v1.7.md

```diff
@@ -32,6 +32,8 @@ This is a relatively small behavior change, but worth calling out in case you no
 - Don't add a `freshness:` block.
 - Explicitly set `freshness: null`
 
+From v1.7 on, when `dbt deps` is run, it will create/update the `package-lock.yml` file in the _project_root_ where `packages.yml` is recorded. The `package-lock.yml` file contains a record of all packages installed and, if subsequent `dbt deps` runs contain no updated packages in `depenedencies.yml` or `packages.yml`, dbt-core installs from `package-lock.yml`. 
```

Comment:
```suggestion
From v1.7 on, when [`dbt deps`](/reference/commands/deps) is run, it will create/update the `package-lock.yml` file in the _project_root_ where `packages.yml` is recorded. The `package-lock.yml` file contains a record of all packages installed and, if subsequent `dbt deps` runs contain no updated packages in `depenedencies.yml` or `packages.yml`, dbt-core installs from `package-lock.yml`. 
```